fabtests/include/shared.h

```diff
@@ -78,11 +78,11 @@ struct test_size_param {
 	int enable_flags;
 };
 
-extern struct test_size_param test_size[];
-extern const unsigned int test_cnt;
+extern struct test_size_param * test_size;
```

Comment:
* should be next to variable name

Reply:
github changed the comment when submitting to make it a bulleted list.  The '*' should be next to the variable name -- *test_name.

Reply:
> See comments on first patch. Thanks! General comment -- save typing - avoid the space bar. :)

I have made the changes requested. However, I do think it is quite some manual work both for reviewer and developer to inspect code style changes. Many C/C++ based libraries offer python scripts that serve two purposes (1) mark the CI as failed if there is a code styling violation and (2) re-format the developer code into appropriate code style. The check_style.py script used by NS-3 is a nice example that does both https://www.nsnam.org/doxygen/group___check_style.html